test/Microsoft.AspNetCore.WebSockets.ConformanceTest/AutobahnTests.cs

```diff
@@ -7,13 +7,16 @@
 using Microsoft.AspNetCore.Server.IntegrationTesting;
 using Microsoft.AspNetCore.Testing.xunit;
 using Microsoft.AspNetCore.WebSockets.ConformanceTest.Autobahn;
+using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Testing;
 using Xunit.Abstractions;
 
 namespace Microsoft.AspNetCore.WebSockets.ConformanceTest
 {
     public class AutobahnTests : LoggedTest
     {
+        private static readonly TimeSpan TestTimeout = TimeSpan.FromMinutes(10);
```

Comment:
I'm not a fan of bumping up timeout when we run into spurrious test, I think it hides real bugs. I think the question to ask ourselves is "would a user think this timeout is a reasonable timeframe for this scenario?". If the answer is no then the timeout is probably too high, and I'm going to guess that 10 minutes (and maybe 5) is too high.